docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2795 @@
+       "      <use xlink:href=\"#IBMPlexSans-64\" x=\"242.099945\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-69\" x=\"300.09993\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-73\" x=\"325.099915\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-74\" x=\"373.799911\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-72\" x=\"408.899902\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-69\" x=\"445.599899\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-62\" x=\"470.599884\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-75\" x=\"528.599869\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-74\" x=\"585.399857\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-69\" x=\"620.499847\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-6f\" x=\"645.499832\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-6e\" x=\"701.499817\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"pcda3045c21\">\n",
+       "   <rect x=\"51.968438\" y=\"7.2\" width=\"390.6\" height=\"271.8\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.visualization import plot_histogram\n",
+    "plot_histogram(result.quasi_dists+[ideal_distribution],\n",
+    "               bar_labels=False,\n",
+    "               legend=['optimization_level=0',\n",
+    "                       'optimization_level=3',\n",
+    "                       'optimization_level=3 + dd',\n",
+    "                       'ideal distribution'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can confirm this by computing the [Hellinger fidelity](https://qiskit.org/documentation/stubs/qiskit.quantum_info.hellinger_fidelity.html) between each set of results and the ideal distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.891\n",
+      "0.957\n",
+      "0.967\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.quantum_info import hellinger_fidelity\n",
+    "for counts in result.quasi_dists:\n",
+    "    print(\n",
+    "        f\"{hellinger_fidelity(counts, ideal_distribution):.3f}\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'0.3.0'"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import qiskit_ibm_runtime\n",
+    "qiskit_ibm_runtime.version.get_version_info()"
```

Comment:
0.4.0 was released this morning, could you please run this cell one more time and then we are good.